src/libparms/db/parmsdbappstate.cpp

```diff
@@ -124,6 +127,14 @@ bool ParmsDb::insertAppState(AppStateKey key, const std::string &value) {
     int errId = 0;
     std::string error;
     bool found = false;
+    std::string valueStr = value;
+    if (valueStr == "") {
+        LOG_WARN(_logger, "Value is empty for AppStateKey: " << static_cast<int>(key));
```

Comment:
Nice to have: converter to display keys as strings in logs.

Reply:
Here it is 👍🏾 [6282f88](https://github.com/Infomaniak/desktop-kDrive/pull/198/commits/6282f883a184d46922224bfee0d6b3dc30f2a401)